src/repos/baseRepo.js

```diff
@@ -0,0 +1,44 @@
+const { EntityIsNotDefinedError, NotFound } = require('./common')
+
+class BaseRepo {
+    static get entity() {
+        throw new EntityIsNotDefinedError()
```

Comment:
Could you please clarify how would code ever go here if the behaviour of `get` is rewritten in the User/Room (to return the appropriate class)?

Reply:
This is kinda an abstract method. It force developers to overwrite this method defining correct entity class. It raise the exception if it is not overwritten.

Reply:
Ok.
Sorry but why have it in the `baseRepo` then?

Reply:
let imagine you created a new repo. ChatRepo for instance, write some code and forgot to define entity. When you test that code, you see appropriate error message instead of `undefined is not a function` 
`Entity type is not defined for ChatRepo`. It is easier to find what is going on.